addons/l10n_ar_withholding/data/template/account.tax-ar_ex.csv

```diff
@@ -0,0 +1,9 @@
+id,description,name,active,sequence,amount_type,amount,tax_group_id,type_tax_use,l10n_ar_withholding,l10n_ar_withholding_amount_type,repartition_line_ids/repartition_type,repartition_line_ids/document_type,repartition_line_ids/account_id,description@es,name@es
+ex_tax_retencion_suss_sufrida,WTH SUSS I,SUSS Withholding incurred,True,10,fixed,0,tax_group_other_withholding,none,customer,untaxed_amount,base,invoice,,Ret SUSS S,Retención SUSS Sufrida
+,,,,,,,,,,,tax,invoice,base_retencion_suss_sufrida,,
+,,,,,,,,,,,base,refund,,,
+,,,,,,,,,,,tax,refund,base_retencion_suss_sufrida,,
+ex_tax_retencion_ganancias_sufrida,WTH Earnings I,Retención Earnings incurred,True,4,fixed,0,tax_group_withholding_earnings,none,customer,untaxed_amount,base,invoice,,Ret Ganancias S,Retención Ganancias Sufrida
```

Comment:
@maq-adhoc quick feedback, this is still half ESP half ENG 😅

Reply:
DONE!